services/vision/mlvision/classifier.go

```diff
@@ -45,9 +45,15 @@ func attemptToBuildClassifier(mlm mlmodel.Service) (classification.Classifier, e
 			return nil, err
 		}
 
-		probs := unpack(outMap, "probability")
+		probs, err := unpack(outMap, "probability")
+		if err != nil {
+			return nil, err
+		}
 		if len(probs) == 0 {
-			probs = unpack(outMap, DefaultOutTensorName+"0")
+			probs, err = unpack(outMap, DefaultOutTensorName+"0")
+			if err != nil {
+				return nil, err
+			}
```

Comment:
Would you ever get to this statement? If probability was not able to be unpacked, wouldn't it just return the error and not come to this statement?